packages/modules/src/modules/banners/designableBanner/DesignableBanner.tsx

```diff
@@ -107,6 +107,11 @@ const DesignableBanner: ReactComponent<BannerRenderProps> = ({
     const { basic, primaryCta, secondaryCta, highlightedText, closeButton, ticker } =
         design.colours;
 
+    const { abTestName, abTestVariant } = tracking;
+
+    const showMobilePaymentIcons =
+        abTestName === 'MOBILE_PAYMENT_ICONS' && abTestVariant === 'SHOW-MOBILE-PAYMENT-ICONS';
```

Comment:
Let's use: `abTestName.includes('MOBILE_PAYMENT_ICONS')`
The test names are likely to be like:
`2024-01-01_MOBILE_PAYMENT_ICONS_UK`

Reply:
also, let's keep formatting consistent - 'SHOW_MOBILE_PAYMENT_ICONS'